### DISCUSSION TITLE
cant appload photo proof student ID

### DISCUSSION THREAD

**Discussion by dlugg** [community/community#91157] [GitHub Education]
### Hi everyone,

When I press select rear camera it doesn't give me anything to take a picture, no such problem with front camera. I tried from different devices and different browsers, same problem 

![8F8305C1-E4A2-4BE9-B6F0-B6E62BFD2D98](https://github.com/community/community/assets/155900687/4fafd9ba-488c-443a-86e2-17c8a340ca9f)

**Comment by ehb54** [comment]
I have the same issue, the rear facing camera, even though selected, does not work, it always uses the front facing camera, which makes it near impossible to take a proper picture. Tried two different android devices.

**Comment by kaushal878** [comment]
yes, I have the same issue, the rear-facing camera, even though selected, does not work, it always uses the front-facing camera, which makes it nearly impossible to take a proper picture, So I try to upload the option in below but try to click original photo without edited and al try not to be too perfect photo.  so I suggest uploading Enrollment
![WhatsApp Image 2024-01-24 at 13 05 51_87698dfb](https://github.com/community/community/assets/46087984/75d2ae4a-fda6-4a17-8dd1-9c89294d6339)
 letter
